bl4ck4ndbr0wn/iReporter
# #162084003 user can create red flag record

#### What does this PR do?
Allows user to create a Create a red-flag record.
#### Description of Task to be completed?
1. must have a simple descriptive form title and description
2. must have image and video upload link
3. must have a geo location tag on report creation.
#### How should this be manually tested?
run the dev environement 
``` 
cd UI && npm i && npm run server
```
navigate to homepage on `localhost:3000` then click create-red flag link then a form page will appear with direction on how to fill it.
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
#162084003
#### Screenshots (if appropriate)
![create flag 1](https://user-images.githubusercontent.com/33542127/48816879-f9df5900-ed54-11e8-90b7-0daf7f4509e3.png)
![create flag 2](https://user-images.githubusercontent.com/33542127/48816882-fc41b300-ed54-11e8-8030-da4f520d8541.png)

#### Questions:
